app/controllers/internal/users_controller.rb

```diff
@@ -106,6 +106,15 @@ def send_email
     end
   end
 
+  def verify_email_ownership
+    if VerificationMailer.account_ownership_verification_email(params).deliver
+      flash[:success] = "Email Verification Mailer sent!"
+      redirect_back(fallback_location: "/users")
```

Comment:
`fallback_location: users_path`, though I have a question, if this is in the internal admin, why is it falling back to https://dev.to/users which is not a real page?